### DISCUSSION TITLE
Data cannot be loaded

### DISCUSSION THREAD

**Discussion by RobertArustamyan** [community/community#112279] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

import json

def read_load_json_file(filename, page_number):
    '''
    Reads file and returns data by page (data is sorted by page)
    :param filename: Name of file where is data
    :param page: number of wanted page
    :return: return json format data
    '''
    with open(filename, 'r') as file:
        data = json.lod(file)
    page = f'page{page_number}'
    return d[page]

**Comment by RobertArustamyan** [comment]
data is not loaded

**Comment by SergeyArzumanyan** [comment]
you're missing an 'a'

**Comment by ghostinhershell** [comment]
Hi @RobertArustamyan and @SergeyArzumanyan, 

Thanks so much for visiting our community. When reviewing posts in the community I noticed that there seems to be some coordination between both of you that may or may not be done with the intent of earning the Galaxy Brain badge for answered posts.

I want to note that, because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, **we’ve made the difficult decision to disable the ability to earn Achievements in our Community**.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Users who continue to create low quality, nonsensical and spam posts in order to earn Achievements in the community can and will face:

- Content Removal
- Content Blocking
- Community access temporarily or permanently blocked
- GitHub Account Suspension
- GitHub Account Termination

Let us know if you have any other questions 🚀
